incubator/hnc/e2e/tests/delete-anchor-crd_test.go

```diff
@@ -0,0 +1,68 @@
+package test
+
+import (
+	"os/exec"
+	"os"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("Delete-anchor-crd", func() {
+	It("should create parent and deletable child, and delete the CRD", func() {
+		// create a namespace delete-crd-parent
+		cmd := exec.Command("kubectl", "create", "ns", "delete-crd-parent")
+		stdout, err := cmd.CombinedOutput()
+		GinkgoT().Log(string(stdout))
+		Expect(err).Should(BeNil())
+
+		// create a namespace delete-crd-child and set its parent as delete-crdparent
+		cmd = exec.Command("kubectl", "hns", "create", "delete-crd-child", "-n", "delete-crd-parent")
+		stdout, err = cmd.CombinedOutput()
+		GinkgoT().Log(string(stdout))
+		Expect(err).Should(BeNil())
+
+		// allow cascading delete for delete-crd-child
+		cmd = exec.Command("kubectl", "hns", "set", "delete-crd-child", "-a")
```

Comment:
Why do we need to set `allowCascadingDelete` here? Seems it's irrelevant here?

Reply:
I put that in the original test. Hmm. I didn't comment _why_ I did that (d'oh!) but if I had to guess, I wanted to make it as easy as possible for HNC to delete it, and then make sure that it didn't.

But now that you don't need `allowCascadingDelete` to delete a leaf subnamespace anyway, I think we can delete this line too.

Reply:
Removed